### ISSUE TITLE
Spectrum not recognizing matplotlib?

### ISSUE THREAD

**Issue by bretthandrews** [marvin-manga/marvin_public_old#3]
Renbin: 

I think i have successfully got marvin installed. However, when I try to plot, it complains about 'matplotlib' not being installed. But clearly it can be imported so I don't understand why. 

Here is the screen shot. Any idea?  I am using ipython from anaconda. 
![renbin_matplotlib_importerror](https://cloud.githubusercontent.com/assets/12723826/16541271/86652b74-404c-11e6-8f1d-e457298f2377.png)

**Comment by bretthandrews** [comment]
Renbin,

I'm not sure what's going on here. Perhaps try 

`import matplotlib.pyplot as pyplot`

since that is the exact call that the script is making.

**Comment by bretthandrews** [comment]
We could also try changing the block that checks whether pyplot has been imported to
`if pyplot not in sys.modules`

**Comment by albireox** [comment]
Yes, the first thing to try is to check whether Renbin can import `import matplotlib.pyplot as pyplot`. I'd be surprised if you can have matplotlib installed without pyplot, but maybe anaconda has a weird installation ...
